packages/doctor/lib/general.js

```diff
@@ -5,9 +5,21 @@ import NodeDetector from './node-detector';
 import {util} from '@appium/support';
 import {EOL} from 'os';
 import '@colors/colors';
+import {resolveAppiumHome} from '@appium/support/build/lib/env';
 
 let checks = [];
 
+class AppiumHomeCheck extends DoctorCheck {
+  async diagnose() {
+    return ok(`Default APPIUM_HOME is ${await resolveAppiumHome()}`);
+  }
+
+  fix() {
+    return;
+  }
```

Comment:
Do we need this method at all?  If `DoctorCheck#fix()` does something we want to avoid, it can just be empty:

```suggestion
  fix() {}
```